### PR TITLE
Changing the display of scenarios

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

File: app/views/scenarios/_outputs.html.haml [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
+    .card-header 🤖 #{link_if_variable(name)}
+    .card-body
+      - if value.class == Array
+        %ul.list-group.list-group-flush
+          - value.each_with_index.map do |item, key|
+            %li.list-group-item
+              %strong
+                -#assume this is for a person
+                = @scenario.input_persons.keys[key]
+              .pull-right=item.to_json
```
[the remark this answers is not in the document]
Comment (Br3nda) [reply app/views/scenarios/_outputs.html.haml]:
```suggestion
              .pull-right= item.to_json
```

File: app/views/scenarios/_outputs.html.haml [hunk 2]
```diff
@@ -1,25 +1,21 @@
...
+- @scenario.outputs.each do |name, value|
+  .card
+    .card-header 🤖 #{link_if_variable(name)}
+    .card-body
+      - if value.class == Array
+        %ul.list-group.list-group-flush
+          - value.each_with_index.map do |item, key|
+            %li.list-group-item
+              %strong
+                -#assume this is for a person
```
[the remark this answers is not in the document]
Comment (Br3nda) [reply app/views/scenarios/_outputs.html.haml]:
```suggestion
                -# assume this is for a person
```

File: app/views/scenarios/_inputs.haml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    = render 'input_card', name: property_name, icon: '🏠', attributes: attributes
+
+- if @scenario.other_input_keys.size.positive?
+  .card
+    .card-body
+      %ul.list-group.list-group-flush
+        - @scenario.other_input_keys.each do |key|
+          %li.list-group-item
+            %strong= link_if_variable(key)
+            .pull-right= @scenario.inputs[key].to_json
```
[the remark this answers is not in the document]
Comment (Br3nda) [reply app/views/scenarios/_inputs.haml]:
```suggestion
            .pull-right= @scenario.inputs[key].to_json

```

File: app/views/scenarios/show.html.haml [hunk 4]
```diff
@@ -1,42 +1,11 @@
...
-    %h3 Variables used in this scenario:
-  .card
-    .card-body
-      %ul
-        - @variables.each do |v|
-          %li
-            - if v.namespace.present?
-              %span.badge= v.namespace
-            = link_to v.description, v
+  %h1.display-3= @scenario.name
```
Comment (dlouise64) [inline app/views/scenarios/show.html.haml]: @Br3nda what does `.display-3` do? Also on percy the Families heading needs margin above it.

Comment (Br3nda) [reply app/views/scenarios/show.html.haml]: Display makes it bigger. It's part of bootstrap.

**Pull request merged**